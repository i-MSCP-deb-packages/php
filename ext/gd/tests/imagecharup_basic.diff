001+ f893ac9f9525d176ce543a6325c7bf13
001- 79b48d5cef6d489bb68573df0296d775