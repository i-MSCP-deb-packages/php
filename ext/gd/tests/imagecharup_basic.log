
---- EXPECTED OUTPUT
79b48d5cef6d489bb68573df0296d775
---- ACTUAL OUTPUT
f893ac9f9525d176ce543a6325c7bf13
---- FAILED
