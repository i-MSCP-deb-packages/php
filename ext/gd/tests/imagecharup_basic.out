f893ac9f9525d176ce543a6325c7bf13